src/screen/DashboardScreen.tsx

```diff
@@ -77,12 +77,14 @@ export function DashboardHeaderLeft() {
 
 function DashboardTitle({setNetworkSelectOpen}: {setNetworkSelectOpen: (v: boolean) => void}) {
   const {currentNetwork} = useContext(NetworkContext);
-  const {status} = useContext(ChainApiContext);
+  const {status} = useApi();
 
   return (
     <TouchableOpacity style={styles.titleContainer} onPress={() => setNetworkSelectOpen(true)}>
       <Text category="s1">Litentry</Text>
-      {currentNetwork ? <NetworkItem item={currentNetwork} isConnected={status === 'ready'} /> : null}
+      {currentNetwork ? (
+        <NetworkItem item={currentNetwork} isConnected={status === 'connected' || status === 'ready'} />
```

Comment:
We could do this check in the hook if you think we would need to check this in other places:

`const {isReady} = useApi();`

Reply:
For now it's the only use case, I will keep this in mind if in future we were repeating this somewhere else.